docs/master/security/authentication.md

```diff
@@ -54,6 +54,16 @@ extend type Query @guard(with: ["api:admin"]) {
 }
 ```
 
+### Using Laravel Sanctum
+
+If you are using [Laravel Sanctum](https://laravel.com/docs/master/sanctum) to authenticate users, specify `sanctum` middleware.
```

Comment:
```suggestion
Lighthouse will use your application's default guard, unless specified otherwise.
If you are using [Laravel Sanctum](https://laravel.com/docs/master/sanctum) only for your API,
you can set it when using `@guard`:
```

Reply:
It is not necessary to specify `sanctum` if it is the default.

Reply:
Sorry, I misunderstood the authentication mechanism.
In `config/auth.php`, setting `guards.api.driver` to `sanctum` solved.

Anyway thank you  for the suggestion.